packages/baseui-codemods/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@uber-web-ui/baseui-codemods",
-  "version": "1.0.6",
+  "version": "1.0.8",
```

Comment:
should this jump from 6 to 8?

Reply:
i already published the change :) (took 2 tries)